SECURITY.md

```
@@ -0,0 +1,172 @@
+- Local denial of service
+- **Response Time**: Within 1 week
+
+### **Low (CVSS 0.1-3.9)**
+- Minor information leaks
+- Low-impact vulnerabilities
+- **Response Time**: Next regular release cycle
+
+## 📞 Contact Information
+
+### **Security Team**
+- **Primary Contact**: security@no-hands-company.com
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)